A83-xds-gcp-authn-filter.md

```diff
@@ -283,9 +283,88 @@ a cache size change will wind up affecting the old filter instance,
 which in principle it shouldn't, but that is considered acceptable for
 this type of change.
 
-##### Java and Go
+##### Java
 
-TODO(sergiitk, ejona86, dfawley): Fill this in.
+In Java, filter state will be retained by the xDS HTTP Filter objects themselves.
+
+The GCP Authentication filter will store the call credentials cache as a regular
+field on a `GcpAuthenticationFilter` object. No in-filter logic will be needed
+to separate caches per filter instance name, as Java's implementation
+will produce distinct `GcpAuthenticationFilter` instances, and therefore,
+different caches.
+
+To achieve this, we need to make several key changes to the class design.
+
+**Phase 1. Introduce Filter.Provider**
+
+Implemented in: https://github.com/grpc/grpc-java/pull/11883.
```

Comment:
Implementation links don't belong in this section; please move them down to the "Implementation" section below (see the list of Java PRs starting on line 466).